src/index.js

```diff
@@ -0,0 +1,73 @@
+import './style.css';
+import ToDoList from './modules/ToDoList.js';
+import Task from './modules/Task.js';
+import createTaskDOM from './modules/taskEvents.js';
+import * as storage from './modules/localStorageFunctions.js';
+
+const toDoList = new ToDoList();
+
+toDoList.taskList = storage.load('tasks');
+
+// test variables if the list is empty
+// toDoList.addTask(new Task('task 1', 3));
+// toDoList.addTask(new Task('task 2', 2));
+// toDoList.addTask(new Task('task 3', 1));
+// toDoList.addTask(new Task('task 4', 0));
```

Comment:
- [ ] For **[best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)** and clean code, kindly make sure you remove all the commented code lines from your code base.